app/1.0/docs/release-notes.md

```diff
@@ -11,6 +11,108 @@ title: Release notes
+    `dom-repeat` instances.
+
+    ```html
+    <template is="dom-repeat" notify-dom-change>
+      ...
+    </template>
+    ```
+
+*   Suppress binding notifications.
+
+    Usage:
+
+    ```js
+    Polymer = { suppressBindingNotifications: true }
+    ```
+
+    Disables notify effects when propagating data _downward_ via bindings. Generally these are not
+    useful unless you are explicitly adding a binding and a change event listener on the same
+    element:
+
+    ```html
+    <my-el foo="{{foo}} on-foo-changed="handleFoo"></my-el>
+    ```
+
+    With this binding, when the host changes the value of `foo`, its `handleFoo` method is invoked
+    to handle the change event. This pattern is generally unnecessary since the host element doing
+    the binding can use a `foo` observer.
+
+    With `suppressBindingNotifications: true`, the event isn't fired  when the host changes the
+    value (but is still fired if `my-el` changes its `foo` value internally.)
+
+    We attempted to make this the default some time back but needed to revert it when we found via
+    [issue #3077](https://github.com/Polymer/polymer/issues/3077) that users were indeed doing this
+    in corner-case scenarios.  Users that avoid these patterns can enjoy the potentially significant
+    benefit of suppressing unnecessary events during downward data flow by opting into this flag.
+
+### `disable-upgrade` attribute
+
+This release adds a `disable-upgrade` attribute API for all Polymer elements when global opt-in
+setting `disableUpgradeEnabled` is true.  This is a lightweight feature useful for performance
+tuning an application, giving fine-grained control over individual element instantiation cost.
+
+When an element is marked with the `disable-upgrade` attribute, its `createdCallback` returns
+early without performing any of the normal Polymer element initialization steps (for example,
+stamping template, setting default properties, running observers, and so on).  The element behaves
+like an element that has not had its definition loaded, **except** that it has the correct prototype
+(as such, methods should not be called on the element until its `disable-upgrade` attribute
+has been removed).
+
+Removing the `disable-upgrade` attribute causes the element to boot up, initialize its properties,
+stamp its template, and so on.
+
+Note this feature is implemented as an attribute API only.  There is no corresponding
+`disableUpgrade` property.  As such, any bindings should be via attribute bindings. For example:
+
+```html
+<script src="/bower_components/webcomponentsjs/webcomponents-lite.js"></script>
+<script>
+  Polymer = { disableUpgradeEnabled: true };
+</script>
+<link rel="import" href="/bower_components/polymer/polymer.html">
+
+  ...
+
+<template is="dom-bind">
+
+  ...
+
+  <!-- upgrade my-element when loggedIn == true -->
+  <my-element disable-upgrade$="{{!loggedIn}}"></my-element>
+
+</template>
+```
+
+### Template strip-whitespace is recursive
```

Comment:
Good catch... I was going to say that I prefer not to use code font in headings, but in practice I've been wildly inconsistent in this area. There's one other heading in this section that includes code font, so I'll change it here for consistency...

We should loop back to this and add a note to the STYLE.md doc.